project_resource_calendar/i18n/fr.po

```diff
@@ -707,6 +713,27 @@ msgstr "SKU"
 msgid "Saturday"
 msgstr "Samedi"
 
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:125
+#, python-format
+msgid "Save"
+msgstr "Save"
+
+#. module: project_resource_calendar
+#. openerp-web
+#: code:addons/project_resource_calendar/static/src/js/save_event_calendar_wizard.js:198
+#, python-format
+msgid "Save & New"
+msgstr "Save & New"
```

Comment:
translation?

Reply:
I wasn't originally in the POT file I downloaded from Odoo to add new translations.